images/linux/scripts/installers/dotnetcore-sdk.sh

```diff
@@ -11,6 +11,7 @@ source $HELPER_SCRIPTS/os.sh
 # Ubuntu 20 doesn't support EOL versions
 LATEST_DOTNET_PACKAGES=$(get_toolset_value '.dotnet.aptPackages[]')
 DOTNET_VERSIONS=$(get_toolset_value '.dotnet.versions[]')
+DOTNET_TOOLS=$(get_toolset_value '.dotnet.tools[]')
```

Comment:
Where do we update `toolset.json` for Ubuntu images?

Reply:
Yes I've missed that. Adding it now.